CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(wxcharts
+        VERSION 1.1
```

Comment:
This should be version 0.1.0. At this stage none of the API is set in stone. Keeping the version at 0.y.z allows us to break compatibility according to semver rules.